PyInstaller/loader/rthooks.dat

```diff
@@ -24,4 +24,5 @@
     'usb':        ['pyi_rth_usb.py'],
     'win32com':   ['pyi_rth_win32comgenpy.py'],
     'cryptography': ['pyi_rth_cryptography.py'],
+    'multiprocessing': ['pyi_rth_multiprocessing.py'],
```

Comment:
Ideally the struct would be calculated based on filenames similar to the standard hook filenames.